Tasks/Common/ios-signing-common/ios-signing-common.ts

```diff
@@ -132,6 +132,52 @@ export async function findSigningIdentity(keychainPath: string) {
     }
 }
 
+/**
+ * Check if provisioning profile contains Cloud entitlement 
+ * @param provProfilePath
+ * @returns {boolean} 
+ */
+export async function cloudEntitlement(provProfilePath: string) {
```

Comment:
Should the name include a verb? E.g. `includesCloudEntitlement` or something like that? Also, if this returns a boolean use a return type? E.g. `) : Boolean {

Reply:
I wasn't sure what is the naming convention , I agree that a verb should be added. @madhurig @hashtagchris what are the naming convention in this case?